console/src/components/Error/ErrorBoundary.tsx

```diff
@@ -19,16 +20,19 @@ export interface Metadata {
 export interface ErrorBoundaryProps {
   metadata: Metadata;
   dispatch: (arg: unknown) => Promise<unknown>; // TODO update when Redux is migrated to TS;
+  errorValue: any;
+  requestError: any;
+  requestURL: any;
```

Comment:
We can  try using the correct types for these: 
- errorValue: Error;
- requestError: Error;
- requestURL: string;